tests/ui/const-generics/min_const_generics/macro-fail.rs

```diff
@@ -35,9 +32,7 @@ fn main() {
   }>;
 
   let _fail = Example::<external_macro!()>;
-  //~^ ERROR: type provided when a constant
 
   let _fail = Example::<gimme_a_const!()>;
-  //~^ ERROR unexpected end of macro invocation
-  //~| ERROR: type provided when a constant was expected
+  //~^ ERROR: unexpected end of macro invocation
```

Comment:
while this one correctly silences the other errors, we want to test the other `gimme_a_const` invocations. I think you can just remove this line, considering we have other tests laready emitting the `unexpected end of macro invocation` error

Reply:
yea, except for this one the tests look good to me, excited to try it out on large code bases

Reply:
Moved these parts in `tests/ui/const-generics/min_const_generics/macro-fail-const.rs`.